rocket-chat-android-widgets/src/main/java/chat/rocket/android/widget/helper/UserAvatarHelper.kt

```diff
@@ -0,0 +1,98 @@
+package chat.rocket.android.widget.helper
+
+import android.content.Context
+import android.graphics.Typeface
+import android.graphics.drawable.Drawable
+import chat.rocket.android.widget.AbsoluteUrl
+import com.amulyakhare.textdrawable.TextDrawable
+import java.net.URLEncoder
+
+object UserAvatarHelper {
+
+    /**
+     * Returns the user avatar URI.
+     *
+     * REMARK: This is often a SVG image (Rocket.Chat:server/startup/avatar.js).
+     *
+     * @param hostname The server's hostname.
+     * @param username The username.
+     * @return The user avatar URI.
+     */
+    fun getUri(hostname : String, username: String): String {
+        return "https://" +
+                hostname.replace("http://", "").replace("https://", "") +
+                "/avatar/" +
+                URLEncoder.encode(username, "UTF-8")
+    }
+
+    /**
+     * Returns the user avatar absolute URI.
+     *
+     * REMARK: This is often a SVG image (Rocket.Chat:server/startup/avatar.js).
+     *
+     * @param absoluteUrl The AbsoluteUrl.
+     * @param username The username.
+     * @return The user avatar absolute URI.
+     */
+    fun getAbsoluteUri(absoluteUrl: AbsoluteUrl, username: String): String {
+        val avatarUri = "/avatar/" + URLEncoder.encode(username, "UTF-8")
+        return absoluteUrl.from(avatarUri)
+    }
+
+    /**
+     * Returns a drawable with username initials.
+     *
+     * @param username The username.
+     * @param context The context.
+     * @return A drawable with username initials.
+     * @see getUsernameInitials
+     */
+    fun getTextDrawable(username: String, context: Context): Drawable {
+        val round = (4 * context.resources.displayMetrics.density).toInt()
+
+        return TextDrawable.builder()
+                .beginConfig()
+                .useFont(Typeface.SANS_SERIF)
+                .endConfig()
+                .buildRoundRect(getUsernameInitials(username), getUserAvatarBackgroundColor(username), round)
+    }
+
+    /**
+     * Returns a string with the username initials. For example: username John Doe returns JD initials.
```

Comment:
I think you should change this docs to something that will work. In your method `John Doe` will not become `JD`.